src/imageio.cs

```diff
@@ -306,6 +310,18 @@ interface CGImageProperties {
 		[Field ("kCGImagePropertyGIFUnclampedDelayTime")]
 		NSString GIFUnclampedDelayTime { get; }
 
+		[iOS (13,0)][TV (13,0)][Watch (6,0)][Mac (10,15)]
+		[Field ("kCGImagePropertyGIFCanvasPixelWidth")]
+		NSString GifCanvasPixelWidth { get; }
```

Comment:
All the other fields here use `GIF` and not `Gif`.

Reply:
yes, I'm a bit ambivalent between old stuff and current rules :)

Reply:
Code completion does not care about casing (it's still alphabetical order) so I tend toward keeping the new stuff following the newer rules (i.e. it's not inconsistent in the usage, only the look).